components/02-elements/tab/_tab.scss

```diff
@@ -1,99 +1,130 @@
+$tab__title-font-weight                 : $font-weight-normal !default;
+$tab__title-font-weight--mobile         : $font-weight-bold !default;
+$tab__title-font-weight--active         : $font-weight-bold !default;
+$tab__title-background                  : $bg-color-base !default;
+$tab__title-background--active          : $gray-light !default;
+$tab__title-background--mobile-active   : $color-secondary !default;
+// content
+$tab__content-padding                   : $spacer--large $spacer--extra-large !default;
+$tab__content-padding--mobile           : $spacer--medium $spacer--large !default;
+// icon
+$tab__icon-padding                      : $spacer--medium !default;
+$tab__icon-size                         : 48px !default;
 .tab {
-    position: relative;
-
     @include mq($screen-l) {
         display: flex;
         flex-flow: row wrap;
     }
-
     &__title {
         position: relative;
-        padding: $tab__padding;
-        border-bottom: $tab__title-border;
-        font-weight: 700;
+        padding: $tab__title-padding;
+        border-top: $tab__title-border--mobile;
+        background: $tab__title-background;
+        text-align: center;
+        font-weight: $tab__title-font-weight--mobile;
+        color: $tab__title-color;
         cursor: pointer;
-        text-align: left;
+
+        &:nth-last-child(2) {
+            border-bottom: $tab__title-border--mobile;
+
+            @include mq($screen-l) {
+                border-bottom: $tab__title-border;
+            }
+        }
 
         @include mq($screen-l) {
-            flex-basis: 25%;
+            flex: 1;
             order: -1;
-            margin-right: $tab__margin;
-            background-color: $tab__title-background;
-            border-bottom: 0 none;
-            bottom: -1px;
+            border-top: none;
+            border-bottom : $tab__title-border;
+            font-weight: $tab__title-font-weight;
         }
 
         &--active {
+            border-top: none;
+            border-bottom: none;
+            background: $tab__title-background--mobile-active;
+            color: $tab__title-color-mobile-active;
+
+            &:nth-last-child(2) {
+                border: none;
+
+                @include mq($screen-l) {
+                    border-bottom: $tab__title-border--active;
+                }
+            }
             @include mq($screen-l) {
-                border-color: $tab__title-border-color--active;
-                border-width: $tab__title-border-width--active;
-                border-style: $tab__title-border-style--active;
+                border-bottom: $tab__title-border--active;
                 background-color: $tab__title-background--active;
+                font-weight: $tab__title-font-weight--active;
+                color: $tab__title-color;
                 z-index: 2;
             }
+            .tab__icon {
+                transform: translateY(-50%)rotateX(180deg);
```

Comment:
Will be nice to have a space between transform rules

Reply:
Done